<html>
<head>
<title>TestNG:  LoginTest</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>LoginTest</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>0/2/2</td>
</tr><tr>
<td>Started on:</td><td>Thu Jun 09 01:26:16 CEST 2022</td>
</tr>
<tr><td>Total time:</td><td>19 seconds (19101 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='tests.LoginTests.invalidLoginTest_EmptyUserEmptyPassword()'><b>invalidLoginTest_EmptyUserEmptyPassword</b><br>Test class: tests.LoginTests<br>Test method: Invalid Login Scenario with empty username and password.<br>Parameters: public void tests.LoginTests.invalidLoginTest_EmptyUserEmptyPassword(java.lang.reflect.Method)</td>
<td><div><pre>java.lang.AssertionError: expected:&lt;Bu alan覺n doldurulmas覺 zorunludur.&gt; but was:&lt;WRONG MESSAGE FOR FAILURE!&gt;
	at pages.LoginPage.verifyLoginPassword(LoginPage.java:51)
	at tests.LoginTests.invalidLoginTest_EmptyUserEmptyPassword(LoginTests.java:23)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 29 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1744190758", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1744190758'><pre>java.lang.AssertionError: expected:&lt;Bu alan覺n doldurulmas覺 zorunludur.&gt; but was:&lt;WRONG MESSAGE FOR FAILURE!&gt;
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:115)
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:125)
	at pages.LoginPage.verifyLoginPassword(LoginPage.java:51)
	at tests.LoginTests.invalidLoginTest_EmptyUserEmptyPassword(LoginTests.java:23)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
	at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
	at org.testng.TestRunner.privateRun(TestRunner.java:794)
	at org.testng.TestRunner.run(TestRunner.java:596)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
	at org.testng.SuiteRunner.run(SuiteRunner.java:276)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
	at org.testng.TestNG.runSuites(TestNG.java:1063)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>1</td>
<td>tests.LoginTests@5c33f1a9</td></tr>
<tr>
<td title='tests.LoginTests.invalidLoginTest_InvalidUserNameInvalidPassword()'><b>invalidLoginTest_InvalidUserNameInvalidPassword</b><br>Test class: tests.LoginTests<br>Test method: Invalid Login Scenario with wrong username and password.<br>Parameters: public void tests.LoginTests.invalidLoginTest_InvalidUserNameInvalidPassword(java.lang.reflect.Method)</td>
<td><div><pre>org.openqa.selenium.UnsupportedCommandException: HTTP method not allowed
Command duration or timeout: 9 milliseconds
Build info: version: &apos;4.2.1&apos;, revision: &apos;ac4d0fdd4a&apos;
System info: host: &apos;guille&apos;, ip: &apos;192.168.0.151&apos;, os.name: &apos;Linux&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;5.4.0-110-generic&apos;, java.version: &apos;11.0.15&apos;
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Command: [3f6bd37f-0406-433d-be6b-8f7e146ec5df, getLog {type=browser}]
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 100.0, moz:accessibilityChecks: false, moz:buildID: 20220428192727, moz:debuggerAddress: localhost:34263, moz:geckodriverVersion: 0.31.0, moz:headless: false, moz:processID: 26647, moz:profile: /tmp/rust_mozprofile7OUUl0, moz:shutdownTimeout: 60000, moz:useNonSpecCompliantPointerOrigin: false, moz:webdriverClick: true, pageLoadStrategy: normal, platformName: LINUX, platformVersion: 5.4.0-110-generic, proxy: Proxy(), se:cdp: ws://localhost:34263/devtoo..., se:cdpVersion: 85.0, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: 3f6bd37f-0406-433d-be6b-8f7e146ec5df
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:217)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:169)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:586)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:606)
	at org.openqa.selenium.remote.RemoteExecuteMethod.execute(RemoteExecuteMethod.java:40)
	at org.openqa.selenium.remote.RemoteLogs.getRemoteEntries(RemoteLogs.java:81)
	at org.openqa.selenium.remote.RemoteLogs.get(RemoteLogs.java:77)
	at utils.logs.JSErrorLogs.isLoginErrorLog(JSErrorLogs.java:13)
	at pages.LoginPage.verifyLogError(LoginPage.java:58)
	at tests.LoginTests.invalidLoginTest_InvalidUserNameInvalidPassword(LoginTests.java:14)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace2010856706", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace2010856706'><pre>org.openqa.selenium.UnsupportedCommandException: HTTP method not allowed
Command duration or timeout: 9 milliseconds
Build info: version: &apos;4.2.1&apos;, revision: &apos;ac4d0fdd4a&apos;
System info: host: &apos;guille&apos;, ip: &apos;192.168.0.151&apos;, os.name: &apos;Linux&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;5.4.0-110-generic&apos;, java.version: &apos;11.0.15&apos;
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Command: [3f6bd37f-0406-433d-be6b-8f7e146ec5df, getLog {type=browser}]
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 100.0, moz:accessibilityChecks: false, moz:buildID: 20220428192727, moz:debuggerAddress: localhost:34263, moz:geckodriverVersion: 0.31.0, moz:headless: false, moz:processID: 26647, moz:profile: /tmp/rust_mozprofile7OUUl0, moz:shutdownTimeout: 60000, moz:useNonSpecCompliantPointerOrigin: false, moz:webdriverClick: true, pageLoadStrategy: normal, platformName: LINUX, platformVersion: 5.4.0-110-generic, proxy: Proxy(), se:cdp: ws://localhost:34263/devtoo..., se:cdpVersion: 85.0, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: 3f6bd37f-0406-433d-be6b-8f7e146ec5df
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:217)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:169)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:586)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:606)
	at org.openqa.selenium.remote.RemoteExecuteMethod.execute(RemoteExecuteMethod.java:40)
	at org.openqa.selenium.remote.RemoteLogs.getRemoteEntries(RemoteLogs.java:81)
	at org.openqa.selenium.remote.RemoteLogs.get(RemoteLogs.java:77)
	at utils.logs.JSErrorLogs.isLoginErrorLog(JSErrorLogs.java:13)
	at pages.LoginPage.verifyLogError(LoginPage.java:58)
	at tests.LoginTests.invalidLoginTest_InvalidUserNameInvalidPassword(LoginTests.java:14)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
	at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
	at org.testng.TestRunner.privateRun(TestRunner.java:794)
	at org.testng.TestRunner.run(TestRunner.java:596)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
	at org.testng.SuiteRunner.run(SuiteRunner.java:276)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
	at org.testng.TestNG.runSuites(TestNG.java:1063)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>6</td>
<td>tests.LoginTests@5c33f1a9</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-skipped'>
<tr><td colspan='4' align='center'><b>SKIPPED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='tests.LoginTests.invalidLoginTest_EmptyUserEmptyPassword()'><b>invalidLoginTest_EmptyUserEmptyPassword</b><br>Test class: tests.LoginTests<br>Test method: Invalid Login Scenario with empty username and password.<br>Parameters: public void tests.LoginTests.invalidLoginTest_EmptyUserEmptyPassword(java.lang.reflect.Method)</td>
<td><div><pre>java.lang.AssertionError: expected:&lt;Bu alan覺n doldurulmas覺 zorunludur.&gt; but was:&lt;WRONG MESSAGE FOR FAILURE!&gt;
	at pages.LoginPage.verifyLoginPassword(LoginPage.java:51)
	at tests.LoginTests.invalidLoginTest_EmptyUserEmptyPassword(LoginTests.java:23)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 29 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1200293689", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1200293689'><pre>java.lang.AssertionError: expected:&lt;Bu alan覺n doldurulmas覺 zorunludur.&gt; but was:&lt;WRONG MESSAGE FOR FAILURE!&gt;
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:115)
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:125)
	at pages.LoginPage.verifyLoginPassword(LoginPage.java:51)
	at tests.LoginTests.invalidLoginTest_EmptyUserEmptyPassword(LoginTests.java:23)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
	at org.testng.TestRunner.privateRun(TestRunner.java:794)
	at org.testng.TestRunner.run(TestRunner.java:596)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
	at org.testng.SuiteRunner.run(SuiteRunner.java:276)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
	at org.testng.TestNG.runSuites(TestNG.java:1063)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>2</td>
<td>tests.LoginTests@5c33f1a9</td></tr>
<tr>
<td title='tests.LoginTests.invalidLoginTest_InvalidUserNameInvalidPassword()'><b>invalidLoginTest_InvalidUserNameInvalidPassword</b><br>Test class: tests.LoginTests<br>Test method: Invalid Login Scenario with wrong username and password.<br>Parameters: public void tests.LoginTests.invalidLoginTest_InvalidUserNameInvalidPassword(java.lang.reflect.Method)</td>
<td><div><pre>org.openqa.selenium.UnsupportedCommandException: HTTP method not allowed
Command duration or timeout: 4 milliseconds
Build info: version: &apos;4.2.1&apos;, revision: &apos;ac4d0fdd4a&apos;
System info: host: &apos;guille&apos;, ip: &apos;192.168.0.151&apos;, os.name: &apos;Linux&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;5.4.0-110-generic&apos;, java.version: &apos;11.0.15&apos;
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Command: [3f6bd37f-0406-433d-be6b-8f7e146ec5df, getLog {type=browser}]
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 100.0, moz:accessibilityChecks: false, moz:buildID: 20220428192727, moz:debuggerAddress: localhost:34263, moz:geckodriverVersion: 0.31.0, moz:headless: false, moz:processID: 26647, moz:profile: /tmp/rust_mozprofile7OUUl0, moz:shutdownTimeout: 60000, moz:useNonSpecCompliantPointerOrigin: false, moz:webdriverClick: true, pageLoadStrategy: normal, platformName: LINUX, platformVersion: 5.4.0-110-generic, proxy: Proxy(), se:cdp: ws://localhost:34263/devtoo..., se:cdpVersion: 85.0, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: 3f6bd37f-0406-433d-be6b-8f7e146ec5df
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:217)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:169)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:586)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:606)
	at org.openqa.selenium.remote.RemoteExecuteMethod.execute(RemoteExecuteMethod.java:40)
	at org.openqa.selenium.remote.RemoteLogs.getRemoteEntries(RemoteLogs.java:81)
	at org.openqa.selenium.remote.RemoteLogs.get(RemoteLogs.java:77)
	at utils.logs.JSErrorLogs.isLoginErrorLog(JSErrorLogs.java:13)
	at pages.LoginPage.verifyLogError(LoginPage.java:58)
	at tests.LoginTests.invalidLoginTest_InvalidUserNameInvalidPassword(LoginTests.java:14)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1614238", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1614238'><pre>org.openqa.selenium.UnsupportedCommandException: HTTP method not allowed
Command duration or timeout: 4 milliseconds
Build info: version: &apos;4.2.1&apos;, revision: &apos;ac4d0fdd4a&apos;
System info: host: &apos;guille&apos;, ip: &apos;192.168.0.151&apos;, os.name: &apos;Linux&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;5.4.0-110-generic&apos;, java.version: &apos;11.0.15&apos;
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Command: [3f6bd37f-0406-433d-be6b-8f7e146ec5df, getLog {type=browser}]
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 100.0, moz:accessibilityChecks: false, moz:buildID: 20220428192727, moz:debuggerAddress: localhost:34263, moz:geckodriverVersion: 0.31.0, moz:headless: false, moz:processID: 26647, moz:profile: /tmp/rust_mozprofile7OUUl0, moz:shutdownTimeout: 60000, moz:useNonSpecCompliantPointerOrigin: false, moz:webdriverClick: true, pageLoadStrategy: normal, platformName: LINUX, platformVersion: 5.4.0-110-generic, proxy: Proxy(), se:cdp: ws://localhost:34263/devtoo..., se:cdpVersion: 85.0, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: 3f6bd37f-0406-433d-be6b-8f7e146ec5df
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:217)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:169)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:586)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:606)
	at org.openqa.selenium.remote.RemoteExecuteMethod.execute(RemoteExecuteMethod.java:40)
	at org.openqa.selenium.remote.RemoteLogs.getRemoteEntries(RemoteLogs.java:81)
	at org.openqa.selenium.remote.RemoteLogs.get(RemoteLogs.java:77)
	at utils.logs.JSErrorLogs.isLoginErrorLog(JSErrorLogs.java:13)
	at pages.LoginPage.verifyLogError(LoginPage.java:58)
	at tests.LoginTests.invalidLoginTest_InvalidUserNameInvalidPassword(LoginTests.java:14)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
	at org.testng.TestRunner.privateRun(TestRunner.java:794)
	at org.testng.TestRunner.run(TestRunner.java:596)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
	at org.testng.SuiteRunner.run(SuiteRunner.java:276)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
	at org.testng.TestNG.runSuites(TestNG.java:1063)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>4</td>
<td>tests.LoginTests@5c33f1a9</td></tr>
</table><p>
</body>
</html>